weblate/addons/models.py

```diff
@@ -534,6 +535,14 @@ def store_post_load_handler(sender, translation: Translation, store, **kwargs) -
     )
 
 
+@receiver(post_save, sender=Change)
+def change_post_save_handler(sender, instance: Change, created, **kwargs) -> None:
+    from weblate.addons.tasks import addon_change
+
+    if created:  # ignore Change updates, they should not be updated anyway
+        addon_change(sender, instance, **kwargs)
+
+
```

Comment:
There is also a bulk signal triggered on bulk change creation which needs to be handled:

https://github.com/WeblateOrg/fedora_messaging/blob/710a5b5156631eb24479e9f02e79d754208f3480/weblate_fedora_messaging/models.py#L36-L40